pyanaconda/ui/gui/spokes/lib/cart.glade

```diff
@@ -215,21 +234,85 @@
               <packing>
                 <property name="expand">False</property>
                 <property name="fill">True</property>
-                <property name="position">2</property>
+                <property name="position">3</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkBox" id="secure_boot_box">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="orientation">vertical</property>
+                <property name="spacing">6</property>
+                <child>
+                  <object class="GtkLabel" id="secure_boot_title">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="margin_top">10</property>
+                    <property name="label" translatable="yes">Secure Boot</property>
+                    <property name="xalign">0</property>
+                    <attributes>
+                      <attribute name="weight" value="bold"/>
+                    </attributes>
+                  </object>
+                  <packing>
+                    <property name="expand">True</property>
+                    <property name="fill">True</property>
+                    <property name="position">0</property>
+                  </packing>
+                </child>
+                <child>
+                  <object class="GtkLabel" id="secure_boot_description">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="label" translatable="yes">When Secure Boot is enabled and configured on the machine, firmware will verify the integrity of the Linux kernel during boot. Secure Boot is not supported on IBM z14 and earlier models, therefore choose to disable it if you intend to boot the installed system on such models.</property>
```

Comment:
The same here. Maybe it's just me but the sentence doesn't give me much sense.

Reply:
Fixed.